alerter/alerter.cs

```diff
@@ -1,29 +1,42 @@
 ﻿using System;
+using System.Diagnostics;
 
-namespace AlerterSpace {
-    class Alerter {
+namespace AlerterSpace
+{
+    class Alerter
+    {
         static int alertFailureCount = 0;
-        static int networkAlertStub(float celcius) {
-            Console.WriteLine("ALERT: Temperature is {0} celcius", celcius);
+        static int networkAlertStub(float celsius)

```

Comment:
Stub which is a test code is part of file along with production code. We should keep test code separate from production code. Primary reason being that test code should not be shipped to deployment and if accidentally test code gets executed, then there can be serious issues.